odoo/addons/base/models/ir_qweb.py

```diff
@@ -2431,10 +2431,16 @@ def _get_field(self, record, field_name, expression, tagName, field_options, val
         field_options['tagName'] = tagName
         field_options['expression'] = expression
         field_options['type'] = field_options.get('widget', field.type)
-        inherit_branding = (
-                self.env.context['inherit_branding']
-                if 'inherit_branding' in self.env.context
-                else self.env.context.get('inherit_branding_auto') and record.has_access('write'))
+        if 'inherit_branding' in self.env.context:
+            inherit_branding = self.env.context['inherit_branding']
+        else:
+            has_access = False
+            try:
+                # has_access may raise an exception, e.g. mail.message > _check_access > _get_mail_message_access
+                has_access = record.has_access('write')
```

Comment:
The `has_access` method is supposed to return a boolean without doing a traceback. Please fix the problem at its source, in mail. Even if it means putting a try around the call to `_get_mail_message_access` (I haven't looked into it further).

(moreover `except Exception` would also hide code errors)

A test could be beneficial.